build/release/README.md

```diff
@@ -0,0 +1,69 @@
+# Releasing jQuery
+
+This document describes the process for releasing a new version of jQuery. It is intended for jQuery team members and collaborators who have been granted permission to release new versions.
+
+## Prerequisites
+
+Before you can release a new version of jQuery, you need to have the following tools installed:
+
+- [Node.js](https://nodejs.org/) (latest LTS version)
+- [npm](https://www.npmjs.com/) (comes with Node.js)
+- [git](https://git-scm.com/)
+
+## Release Process
+
+1. Ensure all milestoned issues/PRs are closed, or reassign to a new milestone.
+2. Verify all tests are passing in [CI](https://github.com/jquery/jquery/actions).
+3. Run any release-only tests, such as those in the [`test/integration`](../../test/integration/) folder.
+4. Ensure AUTHORS.txt file is up to date (this will be verified by the release script).
+  - Use `npm run authors:update` to update.
+5. Create draft blog post on blog.jquery.com; save the link before publishing. The link is required to run the release.
+	- Highlight major changes and reason for release.
+	- Add contributor list generated in the below release script.
+	- Add HTML from the `changelog.html` generated in the below release script.
+6. Run a dry run of the release script:
+
+```bash
+BLOG_URL=https://blog.jquery.com/... npm run release -- -d
+```
+7. If the dry run is successful, run the release script:
+
+```bash
+BLOG_URL=https://blog.jquery.com/... npm run release
+```
+
+This will run the pre-release script, which includes checking authors, running tests, running the build, and cloning the CDN and jquery-dist repos in the `tmp/` folder.
+
+It will then walk you through the rest of the release process: creating the tag, publishing to npm, publishing release notes on GitHub, and pushing the updated branch and new tag to the jQuery repo.
+
+Finally, it will run the post-release script, which will ask you to confirm the files prepared in `tmp/release/cdn` and `tmp/release/dist` are correct before pushing to the respective repos. It will also prepare a commit for the jQuery repo to remove the release files and update the AUTHORS.txt URL in the package.json. It will ask for confirmation before pushing that commit as well.
+
+9. Once the release is complete, publish the blog post.
+
+## Stable releases
+
+Stable releases have a few more steps:
+
+1. Close the milestone matching the current release: https://github.com/jquery/jquery/milestones. Ensure there is a new milestone for the next release.
+2. Update jQuery on https://github.com/jquery/jquery-wp-content.
+3. Update jQuery on https://github.com/jquery/blog.jquery.com-theme.
+4. Update latest jQuery version for [healthyweb.org](https://github.com/jquery/healthyweb.org/blob/main/wrangler.toml).
+5. Update the shipping version on [jquery.com home page](https://github.com/jquery/jquery.com).
```

Comment:
I’d use `1.` for all points; the diffs are much better when you need to insert something in the middle. Some standards repos are using this strategy, probably for the same reason.

Reply:
I would normally do that, but with code examples between the numbered list items, the numbers restart. It would work for the first 5 here, but not for any of the below items.

Reply:
That just means they aren't formatted properly - so it's even better to `1`-index as then you uncover those bugs immediately. You should indent the code snippets (the triple backticks should also be indented) and they will be pulled into the list.

This should be done as even though the numbers may look correct, we have multiple `<ol>` elements so semantics of the lists are incorrect.

Reply:
The same applies to the CDN table at the bottom - since it refers to the 7th point on the list, it should exist within that list.

Reply:
But indenting the code block ruins the syntax highlighting, at least in vs code preview. I haven't pushed it to github to check there.

Reply:
I think semantics are more important - as they affect accessibility - than previews in VS Code. I've used indented code blocks in lists on GitHub and it works perfectly. Maybe the indentation was still not right? I can check later if you don't manage to make it work.

No need to push it to GitHub to test, BTW. You can just put it in a comment, switch to the Preview tab and inspect the HTML.

Reply:
Take that back. Indenting code blocks or unordered list elements only ruins the markdown code syntax highlighting, and I must have assumed the preview was broken, but it seems to work. However, there are downsides. It reduces the max line length fairly significantly for code blocks.

Reply:
Seems to be a fair trade-off. I even misnumbered the last item, so this fixes it.

Reply:
GitHub has condensed any unordered list items and put them on one line. I'm seeing if we can fix that with new lines.

Reply:
I don't know if it's actually an accessibility issue, though. The multiple `<ol>` elements were still readable by screen readers. Subsequent ones had a different `start` attribute so the numbers lined up.